CHANGELOG.md

```diff
@@ -16,7 +16,7 @@
 ### Backward-incompatible changes [experimental]
 
 ### Performance enhancements
-* `skbio.tree.nj` wall-clock time was decreased about 66% by vectorizing `skbio.tree._nj._compute_q`. ([#1512](https://github.com/biocore/scikit-bio/pull/1512))
+* `skbio.tree.nj` wall-clock time was decreased about 99% by vectorizing `skbio.tree._nj._compute_q` and avoiding unnecessary recalculations. ([#1512](https://github.com/biocore/scikit-bio/pull/1512), [#1513](https://github.com/biocore/scikit-bio/pull/1513))
```

Comment:
Thanks! I would expect to see further improvements as the matrix size increases. I think for the purpose of the changelog, just mentioning that this improvement was assessed on a 500x500 matrix is sufficient. I think it is useful for the community to have a ballpark figure on performance improvement. This helps users to evaluate if they want to shift to skbio dev or a future release. For instance, a 5% difference may not matter for a downstream package, but a 100% improvement could represent a very dramatic change and motivate a downstream package to bump priority on adopting a different version of skbio.